### ISSUE TITLE
BackToTop - full screen demo broken

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-react#10585]
The full screen demo is not displaying the backToTop component.
https://staging-v6.patternfly.org/components/back-to-top/react-demos/using-name-selector/

I've noticed that the component is also not being displayed on staging-v6.surge.sh, i'm not sure if it's due to the same issue that's broken this demo. A followup issue in org might be necessary.

it also has two different scrollbars, is that intended? 
@srambach @andrew-ronaldson @mcoker

**Comment by mcoker** [comment]
I think the double scrollbars is maybe from `<Page>` as a child of `<DashboardWrapper>`? https://github.com/patternfly/patternfly-react/blob/bf19a673e58b5fa49d6b0320552acce2d016d860/packages/react-core/src/demos/examples/BackToTop/BackToTopNameDemo.tsx#L25-L27

**Comment by andrew-ronaldson** [comment]
I see there is a switch to turn on _Always show BackToTopButton_ but is this demo supposed to show it only after scrolling?
The fake text about overpass should probably be changed since that is no longer accurate.

![Screenshot 2024-06-11 at 9 27 11 AM](https://github.com/patternfly/patternfly-react/assets/28658548/38b39a49-ab8d-46ce-9c8a-5a8b1aa5dcfd)

**Comment by nicolethoen** [comment]
Ah - I was seeking a demo to show me how to make it appear on scroll because that was behavior that seemed broken on org. so if that's not the intention here, i still have to figure out why it's not working on the org side...
